developer-resources/workshops/August-2019-Bucharest-workshop/docs/README.md

```diff
@@ -0,0 +1,127 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+# Egeria Meeting 28th-30th August 2019 
+
+### Attendees
+* Ferd Scheepers
+* Mandy Chessell 
+* David Radley 
+* Nigel Jones 
+* Maryna Strelchuk 
+* Daniela Otelea
+* Mike Nicpan
+* Bogdan-Mihail Sava
+* Liviu-Constantin Popa
+* Ruxandra-Gabriela Rosu
+* Guy Ackermans
+* Cong Chen
+* Raluca Popa
+* Moya Brannan 
+
+### Agenda
+## 28th August 2019
+### Day 1: Deep Dive of Components and Technical Updates
+#### Morning Session: 
+* ING production target for Egeria lineage use case
+* Automated Deployment & Configuration for Egeria:
+  * We have went through the current process from Nigel about how can we do container deployment. 
+  * Nigel has also showed a demo of building and deploying containers from Docker Hub to IBM Kubernetes clusters.
+  * Action Point: Cong will contact the OpenShift Platform team in ING and talk to Nigel about what is the process in ING and how can we add new features from open source side.
+* OMRS find method syntax
+  * We have went through several regular expression examples as well as methods in OMRSRepositoryHelper.java to understand the syntax. Such as method: String getExactMatchRegex(String s). 
+  * It is important for all the OMAS owners to understand it to do the exact search.
```

Comment:
Suggest changing to:

"It is important for all the OMAS owners to *use* it to do an exact search, so that repositories that only supported a limited set of regular expressions can still service such requests."

Reply:
cool thanks!